paddle/fluid/operators/pool_op.cc

```diff
@@ -187,6 +187,12 @@ Pool2dOpMaker::Pool2dOpMaker(OpProto *proto, OpAttrChecker *op_checker)
       "output height and width. False is the default. If it is set to False, "
       "the floor function will be used.")
       .SetDefault(false);
+  AddAttr<bool>(
+      "exclude_mode",
+      "(bool, default True) Whether to exclude the padding cells when "
+      "calculating, but only work when pooling_type is avg. The defalut "
+      "is exclude mode.")
```

Comment:
Can we have more detail description here.
how does exclude_mode play with pooling_type

Should this be called exclude_padding?

Reply:
Update the description and change name to `exclude_pad`, which is more clear.  Thanks very much! @panyx0718